packages/client/config-overrides.js

```diff
@@ -0,0 +1,40 @@
+const { override } = require("customize-cra")
+const cspHtmlWebpackPlugin = require("csp-html-webpack-plugin")
+
+const cspConfigPolicy = {
+  "default-src": ["'none'"],
+  "script-src": [],
+  "style-src": ["'self'", "'unsafe-inline'"],
+  "img-src": ["'self'", "data:", "cid:", "mid:"],
+  "child-src": ["data:"],
+  "font-src": ["'self'"],
+  "media-src": ["'self'", "data:", "cid:", "mid:"],
+  "object-src": ["'none'"],
+  "form-action": ["'none'"]
+}
+
+const cspConfigPolicy_dev = {
+  ...cspConfigPolicy,
+  "connect-src": ["'self'", "ws://localhost:3000"],
+  "script-src": ["chrome-extension:"]
```

Comment:
We use some Chrome extensions for development to introspect React and Apollo state. We need to allow the `chrome-extension:` scheme in the CSP to allow these to load. But we don't need to allow this in production.